app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<h2>Users</h2>
```

Comment:
i've picked out a few issues of spacing and indentation below...go over the code once more to pick such instances out...i'll review the views tomorrow morning too.